packages/@coorpacademy-components/src/template/my-learning/index.js

```diff
@@ -436,6 +446,7 @@ const MyLearning = (props, context) => {
                       skillTitle={skillsLocales[skill]}
                       focus={selectedSkills.includes(skill)}
                       metrics={{
+                        score: skillsInformation[skill].stats.score,
```

Comment:
style: (maybe ) `const {score, courses, questionsAnswered: skillQuestions, ....} = skillsInformation[skill].stats`
before the return. then you can just use shorthand notation `metrics={{score, courses, skillQuestions,...}}`